joomla/joomla-cms
# [4.2] Use the model for edit check

### Summary of Changes
The access level edit check for super admins should use the model to fetch the level.

### Testing Instructions
- Create a user which can log in on the back end with the following permissions for the users component  
![image](https://user-images.githubusercontent.com/251072/164985470-1f44a7fa-b1cc-44ab-9b74-35c9db62ba36.png)
- Log in as that user on the back end
- Open the access level list view in the back end
- Create a new access level without no group assigned
- Save and close that level
- Click on the new access level in the list which should redirect then to the access level form

### Actual result BEFORE applying this Pull Request
All works but a warning is logged:
_Warning: Undefined array key 0 in /administrator/components/com_users/src/Controller/LevelController.php on line 70_

### Expected result AFTER applying this Pull Request
All works and no warning is logged.
